### PR TITLE
task/9599/authorizing next

### PR DESCRIPTION
**Pull request by KhaiCM**
https://edu-redmine.sun-asterisk.vn/issues/9599

### CODE REVIEW

File: app/Http/Controllers/Admin/RoleController.php [hunk 1]
```diff
@@ -4,82 +4,61 @@
...
+	{
+		$this->role = $role;
+		$this->permission = $permission;
+		$this->user = $user;
+	}
+	public function getRole()
+	{
+		$role = Role::all();
+		$permission = Permission::all();
+		return view('backend.roles.index', compact('permission', 'role'));
```
Comment (BunnyPi04) [inline app/Http/Controllers/Admin/RoleController.php:30]: thiếu dòng trắng trc return

File: resources/views/backend/province/show.blade.php [hunk 2]
```diff
@@ -14,15 +14,17 @@
     </div>
     <table>
         <tr>
-            <th>#</th>
+            <th>STT</th>
```
Comment (BunnyPi04) [inline resources/views/backend/province/show.blade.php]: i18n

**Comment by KhaiCM** [comment]
ready

**Comment by KhaiCM** [comment]
ready

**Pull request merged**